microprot/scripts/processing.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+__author__ = "Tomasz Kosciolek"
+__version__ = "1.01b"
+__last_update__ = "12/19/2016"
+
+
+import click
+import sys
+from skbio import io, Sequence
```

Comment:
I learned from Jose that there is a specific ordering in which you should do imports: 1) build in imports like click, sys, ... 1a) a new line 2) third party imports like skbio 2a) a new line 3) modules from the same project (here microprot) 3a) two new lines